docs/Tutorial.md

```diff
@@ -249,11 +249,12 @@ const PostTitle = ({ record }) => {
     return <span>Post {record ? `"${record.title}"` : ''}</span>;
 };
 
+const required = value => (value ? undefined : 'Required');
 export const PostEdit = (props) => (
     <Edit title={<PostTitle />} {...props}>
         <SimpleForm>
             <DisabledInput source="id" />
-            <ReferenceInput label="User" source="userId" reference="users">
+            <ReferenceInput label="User" source="userId" reference="users" validate={[required]}>
```

Comment:
I think you must now extract the validation array, too, due to a redux-form BC break. Ping @djhi

Reply:
Probably but doesn't validate support a single function too ?

`validate={required}`